feature/metis/conversation/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/conversation/ui/post/PostItem.kt

```diff
@@ -133,6 +138,12 @@ internal fun PostItem(
             postStatus = postStatus,
             authorRole = post?.authorRole,
             authorName = post?.authorName,
+            profilePictureData = ProfilePictureData.create(
```

Comment:
If the author is null, the profilePictureData should be null instead of providing wrong data into the constructor.